src/App.js

```diff
@@ -1,25 +1,54 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import { BrowserRouter as Router, Route, Link, Switch } from 'react-router-dom';
+
+import ListItems from './Components/ListItems';
+import Home from './Components/Home';
+import AddItem from './Components/AddItem';
+
 import './App.css';
 
 function App() {
+  const [activeAdd, setActiveAdd] = useState(false);
+  const [activeList, setActiveList] = useState(false);
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="app">
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route exact path="/add">
+            <AddItem />
+          </Route>
+          <Route exact path="/list">
+            <ListItems />
+          </Route>
+        </Switch>
+        <footer>
+          <nav className="links-wrapper">
+            <ul className="links">
+              <li className={`link list-link ${activeList ? 'active' : ''}`}>
+                <Link
+                  to="/list"
+                  onClick={() => (setActiveAdd(false), setActiveList(true))}
+                >
+                  List
+                </Link>
+              </li>
+              <li className={`link add-link ${activeAdd ? 'active' : ''}`}>
+                <Link
+                  to="/add"
+                  onClick={() => (setActiveList(false), setActiveAdd(true))}
+                >
+                  Add Item
+                </Link>
+              </li>
```

Comment:
Really nice implementation that does the job perfectly!🎉 There is a really nice alternative too, using the `NavLink` instead of `Link`.  Specifically meant to be used to apply styles to the current selected link, `NavLink` comes with a prop called `activeClassName` that you pass in a class name and style the class name any how you want.

This is how I used it and it does the same thing, with shorter code.😁
```
<li >
                <NavLink
                  to="/list"
                  activeClassName="active"
                >
                  List
                </NavLink>
              </li>
              <li >
                <NavLink
                  to="/add"
                  activeClassName="active"
                >
                  Add Item
                </NavLink>
              </li>
```
and css: 

```
.active {
  font-weight: bold;
}
```